schema/1.0.0/schema.json

```diff
@@ -0,0 +1,221 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "YAML Schema Validation for Azure Agent Configuration",
+    "description": "JSON Schema for validating the agent YAML configuration",
+    "type": "object",
+    "required": ["version", "name", "model", "instructions", "tools"],
+    "additionalProperties": false,
+    "properties": {
+      "version": {
+        "type": "string",
+        "description": "YAML schema version",
+        "const": "1.0.0"
+      },
+      "name": {
+        "type": "string",
+        "description": "Agent name",
+        "maxLength": 256
+      },
+      "description": {
+        "type": "string",
+        "description": "Agent description"
+      },
+      "metadata": {
```

Comment:
The metadata could be any object, so the "required" and "properties" can be removed.